s3-benchmark.go

```diff
@@ -307,12 +311,16 @@ func main() {
 	object_data_md5 = base64.StdEncoding.EncodeToString(hasher.Sum(nil))
 
 	// Create the bucket and delete all the objects
-	createBucket()
+	createBucket(true)
 	deleteAllObjects()
 
 	// Loop running the tests
 	for loop := 1; loop <= loops; loop++ {
 
+		// reset counters
+		upload_count = 0
+		download_count = 0
+		delete_count = 0
```

Comment:
I was wondering why successive loops were always faster than the previous loop...